docs/matrix.md

```diff
@@ -193,34 +442,35 @@ Consuming `Results` from previous `TaskRuns` or `Runs` in a `Matrix`, which woul
 
 ```yaml
 tasks:
-...
+---
```

Comment:
this line is used to separate multiple objects in one yaml file; I think it's a bit confusing to have within the example pipeline spec.